spec/models/reservation_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Reservation, type: :model do
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:unit) }
+  end
+
```

Comment:
it 'is not valid without a name' do
      category = Group.create(name: 'Example Category', icon:, user:)
      payment = Expense.new(amount: 100.00, author: user, group: category)
      expect(payment).not_to be_valid
      expect(payment.errors[:name]).to include("can't be blank")
    end


    it 'is valid with valid attributes' do
      category = Group.create(name: 'Example Category', icon:, user:)
      payment = Expense.new(name: 'Example Payment', amount: 100.00, author: user, group: category)
      expect(payment).to be_valid
    end